extensions/value_generators/models/generators_test.py

```diff
@@ -27,7 +27,7 @@ def test_copier(self):
         generator = generators.Copier()
         self.assertEqual(generator.generate_value({}, **{'value': 'a'}), 'a')
         self.assertEqual(generator.generate_value(
-            {}, **{'value': 'a', 'parse_with_jinja': False}), 'a')
+            None, **{'value': 'a', 'parse_with_jinja': False}), 'a')
```

Comment:
Can you explain this change?

Reply:
I am sorry about this one, shouldn't have modified the existing test. I kept the old one and added a new one.